include/UIKit/UIView.h

```diff
@@ -379,11 +379,10 @@ UIKIT_EXPORT_CLASS
 @end
 
 // WinObjC Xaml Extensions
-@class WXFrameworkElement;
 @interface UIView (UIKitXamlExtensions)
-+ (WXFrameworkElement*)createXamlElement;
-@property (nonatomic, readonly, strong) WXFrameworkElement* xamlElement;
-- (instancetype)initWithFrame:(CGRect)frame xamlElement:(WXFrameworkElement*)xamlElement;
++ (RTObject*)createXamlElement;
```

Comment:
We can't reference WXFrameworkElement directly if we don't want UIKit to have a dependency on the projection libraries. Instead we'll return a generic RTObject*.

Notice that this is a breaking change for clients. Not only is the return type different, but you cannot downcast from RTObject* to WXFrameworkElement*. The returned object is genuinely not a WXFrameworkElement. To get from the RTObject* to WXFrameworkElement*, clients will have to do a rt_dynamic_cast.